code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -759,7 +759,8 @@
 					/obj/item/reagent_containers/spray,
 					/obj/item/reagent_containers/medigel,
 					/obj/item/reagent_containers/cup/vial, //SKYRAT EDIT ADDITION - HYPOSPRAYS
-					/obj/item/reagent_containers/chem_pack
+					/obj/item/reagent_containers/chem_pack,
+					/obj/item/reagent_containers/hypospray //BUBBER EDIT ADDITION - Hyposprays 2: spray harder
```

Comment:
```suggestion
					/obj/item/reagent_containers/hypospray/medipen, //BUBBER EDIT ADDITION - Hyposprays 2: spray harder
```
I don't see why the hypospray object itself should fit in the fridge, and makes it consistent with the bag.